data/mods/Aftershock/items/armor/modular_exos/exo_gear.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "id": "exo_mounted_drill",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "exoskeleton mounted drill" },
+    "category": "tool",
+    "description": "A powerful header drill with tough carbide bits, this tool mounts to the exoskeleton's arm and extends past the hand.  While turned off, it still makes a powerful mace.",
+    "flags": [ "USE_UPS", "DURABLE_MELEE", "POWERARMOR_COMPATIBLE", "ABLATIVE_LARGE", "EXO_GLOVE_PLATE" ],
+    "weight": "40000 g",
+    "volume": "5 L",
+    "longest_side": "5 cm",
+    "bashing": 60,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "any", "balance": "clumsy" },
+    "material": [ "hardsteel", "vacuum_carbide" ],
+    "symbol": ";",
+    "color": "light_gray",
+    "techniques": [ "WBLOCK_1", "SWEEP" ],
+    "sided": true,
+    "armor": [ { "encumbrance": 20, "coverage": 80, "covers": [ "arm_l", "hand_l" ] } ],
+    "use_action": {
+      "ammo_scale": 0,
+      "menu_text": "Turn on",
+      "type": "transform",
+      "active": true,
+      "need_charges": 1,
+      "msg": "You turn on the drill.",
+      "target": "exo_mounted_drill_on"
+    }
+  },
+  {
+    "id": "exo_mounted_drill_on",
+    "type": "TOOL_ARMOR",
+    "copy-from": "exo_mounted_drill",
+    "name": { "str": "exoskeleton mounted drill (on)", "str_pl": "exoskeleton mounted drills (on)" },
+    "description": "The drill is now rotating and making a low growling noise.  Meant to make short work of the toughest rock, it should work on more mobile surfaces as well.  Use it to turn it off.",
+    "flags": [ "USE_UPS", "MESSY", "POWERED", "DURABLE_MELEE", "DIG_TOOL", "POWERARMOR_COMPATIBLE" ],
+    "bashing": 60,
+    "cutting": 20,
+    "power_draw": 2000,
+    "revert_to": "exo_mounted_drill",
+    "use_action": { "menu_text": "Turn off", "type": "transform", "msg": "You turn off the drill.", "target": "exo_mounted_drill" }
+  },
+  {
+    "id": "exo_hydraulic_gauntlet",
+    "copy-from": "afs_hydraulic_gauntlet",
+    "type": "ARMOR",
+    "name": { "str": "exoskeleton hydraulic gauntlet" },
+    "description": "A huge, heavy metal gauntlet lined with tubing and gauges worn in place of standard exoskeleton gauntlets.  Slow and unwieldy, it uses internal pressure to deliver devastating blows, but takes tremendous strength to use effectively.",
```

Comment:
This might need to be made clear what the exact purpose for this is, maybe somewhat like a construction related? As right now it feels weird but...

 I'd say you could obsolete the old hydraulic gauntlet in aftershock and keep this one but I'd ask the other aftershock maintainers to pitch into this but personally I feel like the old hydraulic gauntlet is a bit bad balance-wise due to the fact its very powerful with no real cost, so having this replace it would be good in my opinion

Reply:
Replacing the original was the intent with this, though I wasn't sure about making it draw additional power. There aren't a lot of non-combat applications, though it could be useful for demolition. Might fit better as a militarized option.